release.md

```diff
@@ -15,51 +15,88 @@ The purpose of this document is to outline a process that standardizes a mechani
+1. On the morning of the scheduled release, review the `web` repository for changes since the last release. Any member of the ShapeShift org can create a release branch, however a CODEOWNER is required to merge it.
+2. Ensure you have the [`gh` CLI tools installed](https://github.com/cli/cli#installation) to automatically create the PR.
+3. Run the following commands.
+    a. `git fetch --tags && git tag` - look for the most recently tagged version deployed to `main` (production).
+    b. Determine the new release number based on semantic versioning. For most releases this is a patch version. For new features use a minor version. E.g. `1.3.0` goes to `1.3.1` or `1.4.0` for a patch or minor version bump respectively.
+    c. Run the command `yarn create-release vX.Y.Z` with your new version.
 4. Get the URL of the ephemeral environment created (can be found in GitHub or on CloudFlare)
-5. Create a thread in the `#operations-publicchat` Discord channel with the following format. `YYYY-MM-DD repo-name releases/vXX.YY.ZZ`. For example `2022-01-18 web releases/v1.55.21`
-6. In the thread create a summary of the release including new customer facing functionality or bug fixes that can help product and operations to verify functionality, ping the following workstreams by mentioning them in the discord thread and identify each of the people filling the above Roles by discord handle. Also include the URL of the ephemeral environment found in the step above. Please also alert any engineer with code shipping to production in the channel.
-    1. Operations
-    1. Engineering
-    1. Product
-7. If blocking issues are discovered in the release branch, it should be reported in the thread created above.
-8. The person who has found the issue should add it to GitHub and comment on the Pull Request with a link to the issue. This will serve as the release checklist to inform the go / no-go decision on the final merge to production.
-9. The Release Manager will be responsible for coordinating any fixes. If they are able to resolve the issue themselves they may, or if not they will pull in the needed engineering resources to do so. Ultimately, they may also decide to push back the release until the next window if the issues are too severe or risky given the time frame.
-10. Blocking issue fixes should be merged directly into the `releases/vXX.YY.ZZ` branch.
-11. Once all blocking issues have been resolved and merged into the release branch, and Operations Lead and Product Lead have signed off, the Release Manager will merge the branch to main, without doing a squash merge.
-12. This needs to be done at the command line (directions to run via script also provided):
-
-    # After testing, merge to main
-
-    For the web codebase, you can use this script: web/scripts/release.sh main vXX.YY.ZZ
-    For other codebases, these are the commands:
+5. Create a thread in the `#operations-publicchat` Discord channel with the following format.
```

Comment:
Might be worth adding for Discord noobs that operations do not actually need to create the thread themselves and there's a button for that under the "+" menu left to text input, TILL (Today I Legit Learn) that anyone can create a thread